diff --git a/node_modules/@angular-devkit/build-angular/src/webpack/configs/common.js b/node_modules/@angular-devkit/build-angular/src/webpack/configs/common.js
index 4702ae9..f986b13 100644
--- a/node_modules/@angular-devkit/build-angular/src/webpack/configs/common.js
+++ b/node_modules/@angular-devkit/build-angular/src/webpack/configs/common.js
@@ -433,7 +433,7 @@ function getCommonConfig(wco) {
                 },
                 {
                     test: /\.m?js$/,
-                    exclude: [/[\/\\](?:core-js|\@babel|tslib|web-animations-js)[\/\\]/, /(ngfactory|ngstyle)\.js$/],
+                    exclude: [/[\/\\](?:core-js|\@babel|tslib|web-animations-js|monaco-editor-core)[\/\\]/, /(ngfactory|ngstyle)\.js$/],
                     use: [
                         ...(wco.supportES2015
                             ? []
